docs/src/openapi-v1.yaml

```diff
@@ -367,6 +367,68 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/AccountValidation'
+  /accounts/{accountId}/convert:
+    get:
+      tags:
+      - accounts
+      summary: Convert a given account Id to a SS58 address.
+      description: Returns the SS58 prefix, the network address format, the SS58 address, the account ID
+        that was given as input parameter, the scheme that was used and if it is a public key or not (boolean).
+      operationId: accountConvert
+      parameters:
+      - name: accountId
+        in: path
+        description: Account ID or Public Key (hex).
+        required: true
+        schema:
+          format: Account ID or Hex
+          type: string
+      - name: scheme
+        in: query
+        description: The cryptographic scheme to be used in order to convert the account ID to 
+          a SS58 address. It can take one of those three values {sr25519, ed25519, ecdsa}.
```

Comment:
```suggestion
          an SS58 address. It can take one of three values: sr25519, ed25519, ecdsa.
```

Reply:
Thank you for all the suggestions and time spent to review! :pray:
I implemented all the other changes in the docs excepting adding the ":" because it thinks it is a key and throws an error while rebuilding the docs.